Download/index.mixer

```diff
@@ -150,18 +150,18 @@ You should start GAP4 using `gap -A -x 80 -r -m 100m -o 1g -K 2g'.
 
 Architecture: x86_64-apple-darwin16.7.0-default64
 
-testing: /Users/alexk/gap-4.9.1/tst/testinstall/DirectProductElement.tst
```

Comment:
Fine. This test output is a sample, not updated each release. You can go further and replace `alexk` by another username of your choice, but only if you wish:)

Reply:
I don't really care, I just found it irritating that the example above refers to GAP 4.10.2 (shown as part of a GAP banner), and then here it is GAP 4.9.1. Actually, the output here likely changed since 4.9.1, but that's OK; I just didn't want to have a glaring discrepancy in here

Reply:
totally agree